docs/sdks/web/GETTING_STARTED.md

```diff
@@ -0,0 +1,82 @@
+## Getting Started
+
+### Add your Web Platform
+For you to init your SDK and interact with Appwrite services you need to add a web platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before and click the 'Add Platform' button.
+
+From the options, choose to add a **Web** platform and add your client app hostname. By adding your hostname to your project platform you are allowing cross-domain communication between your project and the Appwrite API.
+
+### Get Appwrite Web SDK
+#### NPM
+Use Javascript package manager, NPM from your command line to add Appwrite SDK to your project.
+
+```sh
+npm install appwrite
+```
+
+If you're using a bundler (like Browserify or webpack), you can import the Appwrite module when you need it:
+
+```sh
+import * as Appwrite from "appwrite";
+```
+
+#### CDN
+To install with a CDN (content delivery network) add the following scripts to the bottom of your tag, but before you use any Appwrite services:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/appwrite@2.0.0"></script>
+```
```

Comment:
The SDK setup part (`### Get Appwrite Web SDK`) is already covered in the readme layout template.